Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/api/integration/IModIntegrationRegistry.java

```diff
@@ -29,20 +29,19 @@
      * integration.
      * @return Integration instance if exists, null if does not.
      */
-    @Nullable
-    public <T extends IModIntegration> T getIntegration(Class<T> integrationClass);
+    @Nullable <T extends IModIntegration> T getIntegration(Class<T> integrationClass);
```

Comment:
No, you don't put annotation and its target methods in the same line. Please, for the love of gods.